### PR TITLE
Fixes #10569: Adding a json node property by UI make it a string

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/10569

### CODE REVIEW

File: rudder-web/src/main/scala/com/normation/rudder/web/rest/RestExtractorService.scala [hunk 1]
```diff
@@ -636,6 +636,7 @@ case class RestExtractorService (
   private[this] def extractNodeProperty(json : JValue) : Box[NodeProperty] = {
     ( (json \ "name"), (json \ "value") ) match {
       case ( JString(nameValue), value ) =>
+        logger.info(value)
```
Comment (ncharles) [inline rudder-web/src/main/scala/com/normation/rudder/web/rest/RestExtractorService.scala]: it should either be debug, or not there, isn't it?

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by ncharles** [comment]
I cannot add node properties with this change :(

**Comment by ncharles** [comment]
ha, even with branch 4.1 I cannot

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Pull request merged**